src/main/java/vg/civcraft/mc/civchat2/CivChat2Manager.java

```diff
@@ -73,6 +77,24 @@ public CivChat2Manager(CivChat2 pluginInstance) {
 		scoreboardHUD = new ScoreboardHUD();
 	}
 
+	/**
+	 * Used to set the slowmodeTime of global chat.
+	 * @param amount of seconds the slowmode should be set to.
+	 */
+	public static void setSlowmodeTime(Integer time) {
+		if(time < 0) return;
```

Comment:
Bad formatting, have your IDE handle that according to our style guide. Always use brackets around ifs.